### ISSUE TITLE
Simple django-admin UI

### ISSUE THREAD

**Issue by janbaykara** [dabapps/django-db-queue#57]
Would be useful to have a view of `Job`s showing which ones are queued up to run and when, and which ones have been run.

**Comment by jordaneb** [comment]
Hi @janbaykara, thanks for the suggestion.

If you would like a Django admin UI for the Job model, you could register the model in any one of your `admin.py` files in any if your `INSTALLED_APPS` entries.

For example:
```
# yourproject/common/admin.py (assuming INSTALLED_APPS has yourproject.common)
from django.contrib import admin
from django_dbq.models import Job

admin.site.register(Job) 
```

It's maybe better that it's not included by default because it will give the user of the library the ability to customise its behaviour as needed for the project.

**Comment by janbaykara** [comment]
Yes, I suppose it's true that keeping it separate would allow others to customise it. But as a quick copy-paste job, perhaps we could provide this code (and some minor enhancements) as an optional second module for `INSTALLED_APPS`?
